[Docs] Update Tailwind docs to include step about updating popover containers (#33315)
mui/material-ui: docs/data/material/guides/interoperability/interoperability.md

**Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look at the [`StyledEngineProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-styled-engine-sc/src/StyledEngineProvider/StyledEngineProvider.js) implementation in the `@mui/styled-engine-sc` package.

5. Change the target container for `Portal`-related elements so that they are injected under the main app wrapper that was used in point 3. for setting up the `important` option in the Tailwind's config.

```jsx
const root = ReactDOM.createRoot(document.getElementById('root'));
const theme = createTheme({
  components: {
    MuiPopover: {
      defaultProps: {

>         container: root,

Reviewer: ```suggestion
        container: rootElement,
```

      },
    },
    MuiPopper: {
      defaultProps: {

>         container: root,

Reviewer: ```suggestion
        container: rootElement,
```

      },
    },
  },
});

root.render(
  <ThemeProvider theme={theme}>
    <App />
  </ThemeProvider>;
);
```

> **Note:** To see more examples of how to use containers and encapsulation check out the [shadow DOM documentation](/material-ui/guides/shadow-dom/).

Reviewer: I wouldn’t link this here. The solution of the two problems is similar, but it nay be confusing for the people not aware with the shadow dom.

Author: Got it, removed 👍

> **Note:** To see more examples of how to use containers and encapsulation check out the [shadow DOM documentation](/material-ui/guides/shadow-dom/).

Reviewer: ```suggestion
```

**Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look at the [`StyledEngineProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-styled-engine-sc/src/StyledEngineProvider/StyledEngineProvider.js) implementation in the `@mui/styled-engine-sc` package.

> 5. Change the target container for `Portal`-related elements so that they are injected under the main app wrapper that was used in point 3. for setting up the `important` option in the Tailwind's config.

Reviewer 2: ```suggestion
5. Change the target container for `Portal`-related elements so that they are injected under the main app wrapper that was used in step 3 for setting up the `important` option in the Tailwind config.
```
